controller.go

```diff
@@ -155,6 +156,8 @@ func loadSettings() error {
 		Default("false").BoolVar(&nlbCrossZone)
 	kingpin.Flag("nlb-http-enabled", "Enable HTTP (port 80) for Network Load Balancers. By default this is disabled as NLB can't provide HTTP -> HTTPS redirect.").
 		Default("false").BoolVar(&nlbHTTPEnabled)
+	kingpin.Flag("nlb-http-target-port", "Sets the target port for NLB HTTP listener different from --target-port. Requires --nlb-http-enabled.").
+		UintVar(&nlbHTTPTargetPort)
```

Comment:
The new flag enables second target port only for NLB, for ALB both HTTP and HTTPS listeners continue to use single target and port. This could be extended in the same way for ALB by adding another `--alb-http-target-port` flag.

The alternative is to add generic `--http-target-port` flag that would enable second port for HTTP listener for both ALB and NLB.

Reply:
I think `--alb-http-target-port` makes sense to have, too. We can support both and are fine for any use case.

Reply:
I have added `--alb-http-target-port`